zhmcclient_mock/_urihandler.py

```diff
@@ -829,7 +850,7 @@ def get(cls, method, hmc, uri, uri_parms, logon_required):
         for ucpc in console.unmanaged_cpcs.list(filter_args):
             result_ucpc = {}
             for prop in cls.returned_props:
-                result_ucpc[prop] = ucpc.properties[prop]
+                result_ucpc[prop] = prop_copy(prop_copy(ucpc.properties[prop]))
```

Comment:
why copy the copy?

Reply:
oops. one editor change too much. Thanks!